scripts/scripts.js

```diff
@@ -114,6 +151,13 @@ async function loadLazy(doc) {
   const main = doc.querySelector('main');
   await loadBlocks(main);
 
+  document.body.classList.forEach(async (clazz) => {
```

Comment:
It looks like `decoratePageContent` is heavily influencing how the page will look like for gallery pages so maybe it makes sense to load it earlier wdyt ?

Reply:
It's a little bit tricky. I had it called earlier. But I had issues getting getting slideshow to work because the blocks were not fully rendered. I will try to move it again and experiment.

Reply:
You can experiment by slowing down the network eg by setting it to fast 3g and also check the network tab to see how things are loading

Reply:
@icaraps Well I changed it to load block after the template. Please see if works better.